recipe/meta.yaml

```diff
@@ -38,4 +49,5 @@ about:
 
```

Comment:
We've also updated the docs to be https://docs.gitea.com

Reply:
Done